backend/packages/Upgrade/test/integration/ExperimentStats/DetailGroupExperiment.ts

```diff
@@ -185,8 +185,6 @@ export default async function testCase(): Promise<void> {
   // user 2 logs in experiment
   // get all experiment condition for user 2
   experimentConditionAssignments = await getAllExperimentCondition(experimentUsers[1].id, new UpgradeLogger());
-  checkExperimentAssignedIsNull(experimentConditionAssignments, experimentName1, experimentPoint1);
```

Comment:
This function was checking no assignment. How is that getting checked now?

Reply:
Yes. So, this `checkExperimentAssignedIsNull` function was not working as expected due to not matching to the correct DP keys. This function was not supposed to be called for the user2 as this user is not excluded. Instead we can update this to check no assignment for user1, as this user came before enrolling state and would be excluded.

Reply:
Okay, we use the function `checkExperimentAssignedIsNotDefault` to check if the assigned condition is not default condition everywhere in test cases. Will add that.